ofrak_core/test_ofrak/components/test_filesystem_without_xattr.py

```diff
@@ -0,0 +1,18 @@
+import os
+import subprocess
+
+subprocess.run(["pip", "uninstall", "xattr", "-y"])
+
+import pytest
+
+
+@pytest.mark.serial
+async def test_filesystem_without_xattr():
+    cwd = os.getcwd()
+    os.chdir(os.path.dirname(__file__))
+    result = subprocess.run(["pytest", "./test_filesystem_component.py"])
+    assert result.returncode == 0
+    os.chdir(cwd)
+
+
+subprocess.run(["pip", "install", "xattr"])
```

Comment:
I would think the `subprocess.run` calls need to be inside the function to guarantee that they run in the correct order, no? 

Maybe I don't know enough about pytest's execution model, but I would be worried that these might be run when the function is declared, not when it is actually executed by pytest